tests/Client/GetLearnerReportClientTest.php

```diff
@@ -319,6 +319,91 @@ public function testGetLearnerReportProducesCorrectOutputWithoutOptionalInfoSing
         self::assertNull($report->getRoleId());
     }
 
+    /**
+     * Test that Client::getLearnerReport() produces the correct output when
+     * the query does not contain any optional information and the SmarterU API
+     * returns a single LearnerReport.
+     */
+    public function testGetLearnerReportProducesCorrectOutputWithBlankDates() {
+        $accountApi = 'account';
+        $userApi = 'user';
+        $client = new Client($accountApi, $userApi);
+
+        $query = (new GetLearnerReportQuery())
+            ->setEnrollmentId('1')
+            ->setGroupStatus('Active')
+            ->setUserStatus('Active');
+
+        $id = '1';
+        $courseName = 'My Course';
+        $lastName = 'User';
+        $firstName = 'Test';
+        $learningModuleId = '2';
+        $userId = '3';
+        $createdDate = '';
+        $modifiedDate = null;
```

Comment:
I don't think this works. I would actually expect the tag to be absent from `$xmlString` as `null` is perfectly valid CDATA which xml parsers ought to return as a string and not a special null value. There is no null value in xml really. Just elements that don't exist.